docs/en/flight_controller/pixhawk_series.md

```diff
@@ -100,6 +100,23 @@ At very high level, the main differences are:
 
 <a id="licensing-and-trademarks"></a>
 
+### FMUv6 Comparison
+
+| Feature            | **FMUv6X-RT**         | **FMUv6X**        | **FMUv6XC**        |
+| ------------------ | --------------------- | ----------------- | ------------------ |
+| **FMU MCU**        | NXP i.MX RT1176       | STM32H753         | STM32H743V         |
+| **RAM**            | 2 MB                  | 1 MB              | 1 MB               |
+| **Flash**          | 64 MB QSPI            | 2 MB internal     | 2 MB internal      |
+| **IO MCU**         | Internal coprocessor  | STM32F103         | STM32F103          |
```

Comment:
The FMUv6X-RT also supports the STM32F103 coprocessor. Overall the IO MCU is technically optional [ARK FMUv6X PAB](https://docs.px4.io/main/en/flight_controller/ark_pab) doesn't have the IO processor.